src/Ticket.php

```diff
@@ -207,6 +207,20 @@ public function canAssignToMe()
     }
 
 
+    public static function assignToMe($ticket_id, $user_id)
+    {
+        $ticket = new Ticket();
+        if ($ticket->getFromDB($ticket_id)) {
+            if ($ticket->canAssignToMe()) {
```

Comment:
If user is already part of requesters or observers, should it be added to assignees?

Reply:
For now, it adds it (as the Bahaviors plugin), but I can add check ?